applications/swbatch/profile.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "swbatch",
+    "description": "Modules to load for swbatch",
+    "moduleLoads": [
+        {
+            "modulesToLoad": [
+                "swbatch"
```

Comment:
Is `swbatch` a valid module? You can test on the system with: `module spider swbatch`